src/Common/Auth/OpenIDConnect/Repositories/ScopeRepository.php

```diff
@@ -768,19 +774,21 @@ public function getCurrentStandardScopes(): array
                             $scopes_api_portal['patient/' . $scopeRead] = 'patient/' . $scopeRead;
                             break;
                         case 'put':
-                        case 'insert':
+                        case 'create':
                         case 'update':
+                        case 'delete':
                             $scopes_api_portal['patient/' . $scopeWrite] = 'patient/' . $scopeWrite;
                             break;
                     }
                 }
             }
         }
+        $oidc = array_combine($this->oidcScopes(), $this->oidcScopes());
         $scopes_api = array_merge($scopes_api, $scopes_api_portal);
 
         $scopesSupported = $this->apiScopes();
         $scopes_dict = array_combine($scopesSupported, $scopesSupported);
-        $scopesSupported = null;
+        $scopesSupported = null; // this is odd, why do we have this?
```

Comment:
maybe some sort of loop is happening here where scopesSupported is used to build the $scopes_dict